src/iperf_api.c

```diff
@@ -1639,10 +1639,12 @@ iperf_parse_arguments(struct iperf_test *test, int argc, char **argv)
 	    test->use_pkcs1_padding = 1;
 	    break;
 #endif /* HAVE_SSL */
+#if !defined(HAVE_CLOCK_NANOSLEEP) && !defined(HAVE_NANOSLEEP)
 	    case OPT_PACING_TIMER:
 		test->settings->pacing_timer = unit_atoi(optarg);
 		client_flag = 1;
 		break;
+#endif /* !HAVE_CLOCK_NANOSLEEP && !HAVE_NANOSLEEP */
```

Comment:
On the other hand, `--pacing-timer` gets communicated from the client to the server for use in `--reverse` and `--bidir` tests (where the server acts as a sender). So there could be a use case for accepting `--pacing-timer` even on a host that has `nanosleep()` support. I haven't completely thought this through, however.

Reply:
Removed the `#if !defined`, so that now `--pacing-timer` is always available, as per the comments I understand it is needed in two cases:
1. Backward compatibility for scripts that run the tests using this option, so they will not fail on invalid option, even if in practice the value is not used.
2. Per the second comment I realized that the server may need this option.  I changed the help message in `iperf_locale.c` so always show this option, but in case that it is not used by the client say that is only for the server usage in case the client doesn't use it.